generator/github_template/github-workflow.yml

```diff
@@ -0,0 +1,24 @@
+name: Workflow
+
+# trigger when new commits are pushed to master or main branch
+on:
+  push:
+    branches: [ master, main ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install Node.js
+      uses: actions/setup-node@v1
+      with:
+        registry-url: https://registry.npmjs.org/
+    - run: yarn install
+    - run: yarn lint
+    - run: yarn test:unit
+    - run: yarn build
+    - name: Publish Artifact
+      run: yarn publish
+      env:
+        NODE_AUTH_TOKEN: ${{secrets.YARN_TOKEN}}
```

Comment:
It's a common step for a package, but the plugin is used with app. We don’t need to publish it. You can erase these lines. 

The last step for other agents is to save the artifact `/dist` to make it accessible and downloadable. Can you make change in this way ?

Reply:
I misunderstood saving the artifact for publishing to the NPM registry. Let me make the suggested changes.

Reply:
Updated as per your suggestions